cpp/src/dictionary/search.cu

```diff
@@ -79,8 +79,7 @@ struct find_index_fn {
     using ScalarType = cudf::scalar_type_t<Element>;
     auto find_key    = static_cast<ScalarType const&>(key).value(stream);
     auto keys_view   = column_device_view::create(input.keys(), stream);
-    auto iter = thrust::equal_range(thrust::device,  // segfaults: rmm::exec_policy(stream) and
```

Comment:
are you able to reproduce what went wrong here?

Reply:
No, unfortunately not. I was able to trace the comment back to https://github.com/rapidsai/cudf/commit/b273c90d92ada96c22029c2e3be784a55d37aa08, though. @davidwendt do you have any insight into or recollection of this issue? I wasn't able to repro locally or in CI, so I'm assuming that whatever the underlying cause was has since been fixed.

Reply:
https://github.com/NVIDIA/thrust/issues/922
Looks like this may be fixed now.
https://github.com/NVIDIA/thrust/pull/1104